apps/aecore/src/aec_genesis_block_settings.erl

```diff
@@ -1,26 +1,44 @@
 -module(aec_genesis_block_settings).
 
 -export([dir/0,
-         preset_accounts/0]).
+         preset_accounts/0,
+         read_presets/0]).
 
 dir() ->
     filename:join(aeu_env:data_dir(aecore), ".genesis").
 
+-spec preset_accounts() -> list().
 preset_accounts() ->
-    PresetAccountsFile = filename:join([dir(), "accounts.json"]),
-    case file:read_file(PresetAccountsFile) of
-        {error, _Err} ->
+    case ?MODULE:read_presets() of
```

Comment:
Why ?MODULE here... it's the same module anyway?

Reply:
I know this is ugly but I could not find a way around it.
This module is being tested and in order to use `meck` for the `read_presets/1` - a fully qualified function call must be used. Other option would be meck-ing filename module...
I could add a comment for that if you find it useful

Reply:
I guess this is a misunderstanding of meck. You don't need to add the module in the call to make meck pick it up. Ask Adam for details.

Reply:
Just adding this here for posterity: Meck cannot mock local function calls in a module. The alternative is to use a fully qualified call (`Mod:Func(...)`) or even better, not to test using mock functions inside modules but at module borders only.